### DISCUSSION TITLE
v0.3.2

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#64] [Announcements]
**Hej! Jag efterlyser någon som är bra på videoproduktion och vill göra en instruktionsfilm till Privatkopiera. För mer information, se https://github.com/stefansundin/privatkopiera/discussions/63.**

Ändringar i v0.3.2 ([kodändringar](https://github.com/stefansundin/privatkopiera/compare/v0.3.1...v0.3.2)):

- Fixade urplay.

Diskutera nedan.

**Obs:** om du har ett problem som du tror beror på koden och du vill testa en äldre version så kan du, om du använder Firefox, installera en gammal version av tillägget [på den här sidan](https://addons.mozilla.org/en-US/firefox/addon/privatkopiera/versions/). Det finns tyvärr inget enkelt sätt att göra detsamma i Chrome. **Detta hade inte löst problemet med urplay den här gången för att det berodde på en ändring på urplay.**

**Comment by tmagnusson** [comment]
Hm, Privatkopiera verkar inte fungera just nu, varken med Chrome eller Firefox, mot SVTPlay. Jag får bara "Laddar, var god vänta ..." i all evighet. Något ändrat hos SVT? PK v0.3.2 på Linux Mint 20.1.

Edit: funkar på tv4play så det är nog SVT som gjort något ...

**Reply by stefansundin** [reply]
Fixat i v0.3.3.

**Reply by Sniggelurman** [reply]
Hur uppdaterar man till v0.3.3? Trodde det gjordes automatiskt, men omstart av PC (inkl Firefox) etc har inte hjälpt...

**Reply by Bub0** [reply]
Du kan gå hit, https://addons.mozilla.org/sv-SE/firefox/addon/privatkopiera/versions/t
Enklast är att Trycka på ta bort sen lägg till men borde gå att ladda ner och sen öppna också. 🙂

**Reply by stefansundin** [reply]
@Sniggelurman Din webbläsare uppdaterar automatiskt lite då och då. Men om du inte vill vänta på det så är det bara att gå in i listan med dina tillägg och hitta knappen som kollar efter uppdateringar och klicka på den. Notera att ibland så kan det ta några dagar för Chrome att godkänna uppdateringen också, så i de fallen får du vänta lite innan du ens kan hämta ned uppdateringen. För Firefox så godkänns uppdateringarna mycket snabbare.

**Comment by Sniggelurman** [comment]
Stort tack Stefan,

Nu verkar det funka! Jag ska se till att göra en liten donation på din sida. Detta är en toppenfunktion, så det gör jag med glatt hjärta!

En ytterligare fråga: Ser att filformatet ändrats till MKV. Finns det något sätt som jag som användare kan fortsätta med mp4? (Alla TV-apparater accepterar inte MKV).

Med vänlig hälsning
Yngve Svensson (Sniggelurman)

**Reply by stefansundin** [reply]
Hej @Sniggelurman. Du kan ändra filnamnet så att det slutar med `.mp4`. Då läggs också `-absf aac_adtstoasc` till i kommandot, vilket krävs för mp4.

I nuvarande stund så finns det inget val att sätta `.mp4` som default. Får se om det blir en efterfrågad funktion.
